examples/13-1d_holstein/run_afqmc.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+np.random.seed(125)
+from mpi4py import MPI
+
+from ipie.qmc.afqmc import AFQMC
+from ipie.estimators.energy import EnergyEstimator
+from ipie.systems import Generic
+from ipie.addons.eph.hamiltonians.holstein import HolsteinModel
```

Comment:
your module name (eph) does not match the directory name (ephqmc). You're also missing  __init__.py everywhere (toplevel ephqmc/__init__.py, and every new directory you've added.)